server/services/reportService.ts

```diff
@@ -13,11 +13,15 @@ export default class ReportService {
     token: string,
     month: string,
     year: string,
-    reportType: 'lostBeds',
+    reportType: 'applications' | 'lostBeds',
     response: Response,
   ): Promise<void> {
     const client = this.reportClientFactory(token)
 
-    return client.getLostBedsReport(month, year, response)
+    if (reportType === 'lostBeds') {
+      return client.getLostBedsReport(month, year, response)
+    }
+
+    return client.getApplicationsReport(month, year, response)
```

Comment:
Wonder whether this could be a case statement with a default that throws an error, something like

```typescript
return switch(reportType) { 
   case 'lostBeds': { 
       client.getLostBedsReport(month, year, response)
   } 
   case 'applications': { 
      client.getApplicationsReport(month, year, response)
   } 
   default: { 
      throw new Error(`Unknown report type ${reportType}`)
   } 
} 
```

Reply:
Yeah I did debate doing a switch. I'll add it - worth it for the Error